message_system sql.sql

```diff
@@ -8,8 +8,8 @@ CREATE TABLE user (
 	`password` VARCHAR(128) CHARACTER SET utf8 NOT NULL,
 	nickname VARCHAR(256) CHARACTER SET utf8mb4 NOT NULL,
 	email VARCHAR(128) CHARACTER SET utf8 NOT NULL,
-	memo VARCHAR(1024) CHARACTER SET utf8mb4 NOT NULL,
-	location VARCHAR(128) CHARACTER SET utf8 NOT NULL,
+	memo VARCHAR(1024) CHARACTER SET utf8mb4,
+	location VARCHAR(128) CHARACTER SET utf8,
```

Comment:
I would prefer not to use null ever since null will potentially result in NullPointerException. If a user doesn't provide memo and location when registration, just leave them to empty not null. Consider to use `NOT NULL DEFAULT ''`.